src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double base;
+    private final double height;
+
+    public RightTriangle(String color, double base, double height) {
+        super(color);
+        this.base = base;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (height * base) / 2;
```

Comment:
To improve readability, consider using a constant for the number 2. This will make it clear that you are calculating the area of a right triangle using the formula (base * height) / 2.